VinylRecordsRepository.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VinylRecordsManager.Models;
+using VinylRecordsManager.Data;
+using VinylRecordsManager.View;
+using System.Runtime.Remoting.Messaging;
+
+namespace VinylRecordsManager.Data
+{
+    public class VinylRecordsRepository
+    {
+        List<VinylRecords> masterList = new List<VinylRecords>();
+        VinylRecords vinyl = new VinylRecords();
+        VinylRecordsView userEnteredId = new VinylRecordsView();
+        //List here where everything needs to save.
+        public void Create(VinylRecords record)
+        {
+            masterList.Add(record);
+        }
+
+        public void ReadAll()
+        {
+            Console.WriteLine(string.Join<VinylRecords>("\n", masterList));

```

Comment:
The Repo should just return the list to the Controller which will call the View to display. No UI in the Model.